[2025-03-24T12:35:07.580+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-24T12:35:07.754+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:07.860+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [queued]>
[2025-03-24T12:35:07.870+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-24T12:35:07.964+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2024-05-02 06:00:00+00:00
[2025-03-24T12:35:08.017+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3178) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-24T12:35:08.024+0000] {standard_task_runner.py:72} INFO - Started process 3215 to run task
[2025-03-24T12:35:08.051+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2024-05-02T06:00:00+00:00', '--job-id', '138', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpkxflrooj']
[2025-03-24T12:35:08.062+0000] {standard_task_runner.py:105} INFO - Job 138: Subtask ingest
[2025-03-24T12:35:08.264+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [running]> on host 4791d3610749
[2025-03-24T12:35:08.693+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-02T06:00:00+00:00'
[2025-03-24T12:35:08.694+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-24T12:35:08.739+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-24T12:35:08.740+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-24T12:35:08.793+0000] {subprocess.py:99} INFO - Output:
[2025-03-24T12:35:08.808+0000] {subprocess.py:106} INFO - ls
[2025-03-24T12:35:08.816+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-24T12:35:08.890+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-24T12:35:08.891+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2024-05-02T06:00:00+00:00, execution_date=20240502T060000, start_date=20250324T123507, end_date=20250324T123508
[2025-03-24T12:35:09.108+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-24T12:35:09.174+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-24T12:35:09.184+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-25T11:09:24.717+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-25T11:09:24.927+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:25.044+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [queued]>
[2025-03-25T11:09:25.045+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-25T11:09:25.196+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest> on 2024-05-02 06:00:00+00:00
[2025-03-25T11:09:25.233+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=133) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-25T11:09:25.252+0000] {standard_task_runner.py:72} INFO - Started process 167 to run task
[2025-03-25T11:09:25.242+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_postgres_dag', 'ingest', 'scheduled__2024-05-02T06:00:00+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres_dag.py', '--cfg-path', '/tmp/tmpklx1ltx5']
[2025-03-25T11:09:25.258+0000] {standard_task_runner.py:105} INFO - Job 178: Subtask ingest
[2025-03-25T11:09:25.488+0000] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_postgres_dag.ingest scheduled__2024-05-02T06:00:00+00:00 [running]> on host 4c8ab9810a8c
[2025-03-25T11:09:25.909+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion_postgres_dag' AIRFLOW_CTX_TASK_ID='ingest' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-02T06:00:00+00:00'
[2025-03-25T11:09:25.937+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-25T11:09:26.053+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-25T11:09:26.058+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "ls"']
[2025-03-25T11:09:26.089+0000] {subprocess.py:99} INFO - Output:
[2025-03-25T11:09:26.094+0000] {subprocess.py:106} INFO - ls
[2025-03-25T11:09:26.096+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-25T11:09:26.202+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-25T11:09:26.203+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_ingestion_postgres_dag, task_id=ingest, run_id=scheduled__2024-05-02T06:00:00+00:00, execution_date=20240502T060000, start_date=20250325T110924, end_date=20250325T110926
[2025-03-25T11:09:26.402+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-25T11:09:26.468+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-25T11:09:26.481+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
